### PR TITLE
[HOLD] [MOB-933] Remove iOS 9 support for future SDKs

### PR DESCRIPTION
**Pull request by roninopf**

### CODE REVIEW

**Comment by tapashmajumder** [comment]
Did we get approval for doing this from Suhel or Mahavir? We should get a +1 from them.

**Comment by roninopf** [comment]
> Did we get approval for doing this from Suhel or Mahavir? We should get a +1 from them.

Whoops. You're right, we should. It'll be ready for when they say yes, at least. :P

File: Tests/common/CommonMocks.swift [hunk 1]
```diff
@@ -74,7 +73,6 @@ public class MockCustomActionDelegate: NSObject, IterableCustomActionDelegate {
 @objcMembers
 public class MockUrlOpener: NSObject, UrlOpenerProtocol {
     var ios10OpenedUrl: URL?
```
Comment (tapashmajumder) [inline Tests/common/CommonMocks.swift]: We should probably rename to `openedUrl`

**Comment by roninopf** [comment]
Closing for new future PR to drop iOS 9 support

**Pull request closed without merging**